src/db/models/planTag.ts

```diff
@@ -36,6 +36,11 @@ export default defineIDModel({
     optional: {
       comment: { kind: 'checked', type: t.string },
       revisionState: { kind: 'checked', type: PLAN_TAG_REVISION_STATE },
+      type: { kind: 'checked', type: t.string },
```

Comment:
This column looks like a candidate for either enum or a limited set of string values. If you go with enum route, you will need to add the enum in DB migration. `flowObject` is a [good example](https://github.com/UN-OCHA/hpc-api-core/blob/31e17734fc303f1f9fbf4e2005b70a94eb7c47ee/src/db/models/flowObject.ts#L32) that employs both ways, `behavior` column is an enum, whereas `refDirection` and `objectType` have a set of allowed strings.